WordPress/Classes/ViewRelated/Aztec/Stock Photos/StockPhotosPicker.swift

```diff
@@ -36,7 +36,9 @@ final class StockPhotosPicker: NSObject {
         picker.showGroupSelector = false
         picker.dataSource = dataSource
 
-        origin.present(picker, animated: true)
+        origin.present(picker, animated: true) {
+            picker.mediaPicker.searchBar?.becomeFirstResponder()
```

Comment:
This is an open question, and it definitely does not block this PR, but should this be a public method in the picker?